web-animations-2/Overview.bs

```diff
@@ -3043,6 +3094,41 @@ with which it is associated is updated.
 
 </div>
 
+<div class="members">
+:   <dfn dict-member for=KeyframeAnimationOptions>id</dfn>
+::  The string to assign to the generated {{Animation}}'s
+    {{Animation/id}} attribute.
```

Comment:
Again, this part seems to be duplicating level 1 instead of being a delta. Is that intended?